### PR TITLE
DLS-4467: [RP] Play 2.8 library upgrades.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -5,31 +5,35 @@ import sbt._
...
+    "uk.gov.hmrc"                %% s"bootstrap-frontend-$playVersion"  % "5.12.0",
+    "uk.gov.hmrc"                %% "mongo-caching"                     % s"7.0.0-$playVersion",
+    "uk.gov.hmrc"                %% "domain"                            % s"6.1.0-$playVersion",
+    "uk.gov.hmrc"                %% "play-language"                     % s"5.0.0-$playVersion",
+    "org.typelevel"              %% "cats-core"                         % "2.6.0",
+    "com.github.kxbmap"          %% "configs"                           % "0.6.1",
+    "org.julienrf"               %% "play-json-derived-codecs"          % "10.0.2",
+    "com.github.julien-truffaut" %% "monocle-core"                      % "2.0.0",
+    "com.github.julien-truffaut" %% "monocle-macro"                     % "2.0.0",
+    compilerPlugin("com.github.ghik" % "silencer-plugin"    % "1.7.1" cross CrossVersion.full),
```
Comment (sureshhmrc) [inline project/AppDependencies.scala:22]: minor : `1.7.5`

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -5,31 +5,35 @@ import sbt._
...
-    "org.scalatestplus.play"     %% "scalatestplus-play"        % "3.1.2"          % "test, it"
+    "org.scalatest"              %% "scalatest"                 % "3.2.9"               % "test",
+    "org.jsoup"                   % "jsoup"                     % "1.12.1"              % "test",
+    "com.typesafe.play"          %% "play-test"                 % current               % "test",
+    "org.scalamock"              %% "scalamock"                 % "4.2.0"               % "test",
+    "org.scalatestplus"          %% "scalatestplus-scalacheck"  % "3.1.0.0-RC2"         % "test, it",
+    "com.vladsch.flexmark"        % "flexmark-all"              % "0.35.10"             % "test",
+    "uk.gov.hmrc"                %% "reactivemongo-test"        % s"5.0.0-$playVersion" % "test",
+    "com.github.alexarchambault" %% "scalacheck-shapeless_1.14" % "1.2.1"               % "test",
+    "org.pegdown"                 % "pegdown"                   % "1.6.0"               % "test, it",
```
Comment (sureshhmrc) [inline project/AppDependencies.scala:35]: i think you can remove `pegdown` completely

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala [hunk 3]
```diff
@@ -103,7 +107,7 @@ trait ControllerSpec extends WordSpec with Matchers with BeforeAndAfterAll with
               | microservice.upscan-initiate.upscan-store.expiry-time = 1
           """.stripMargin
           )
-        ) ++ additionalConfig
+        ).withFallback(additionalConfig)
```
Comment (sureshhmrc) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala:110]: i think this is not a fallback, this is an additional config, so you should try to use the method that takes additional config

**Comment by sureshhmrc** [comment]
i think you can remove `play.http.filters` config from application.conf ?

File: project/AppDependencies.scala (hunk 1 above)
Comment (rambabu-posa) [reply project/AppDependencies.scala:22]: Somehow, CGT FE is not accepting scala 2.12.13 so cannot use 1.7.5 version here.
Will double check it

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala (hunk 3 above)
Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala:110]: Compiler has suggested to use this function through one warning message.

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**